app/main.py

```diff
@@ -1 +1,22 @@
 # write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        print("Incorrect data!")
+        return
+    if parts[1] == parts[2]:
+        print("Same file names!")
+        return
+    for i, char in enumerate(parts[2]):
+        if char == "/":
```

Comment:
Each OS has differance separotor for path, so you should use os module for working with directories